data/data/powervs/variables-powervs.tf

```diff
@@ -107,19 +107,19 @@ variable "powervs_sys_type" {
 variable "powervs_base_domain" {
   type        = string
   description = "The base domain name of the cluster"
-  default     = ""
+  default     = "openshift-on-power.com"
 }
 
 variable "powervs_cluster_domain" {
   type        = string
   description = "The name of the cluster that all DNS records must belong to."
-  default     = ""
+  default     = "rdr-ipi"
```

Comment:
we should use the base config values to have unique subdomains, otherwise next run will hit the conflicts.

Reply:
cluster_id will be auto generated and it'll be unique

Reply:
yes

Reply:
not really but atleast the one supplied in the install-config.yaml

```shell
$ grep domain terraform.tfvars.json
  "cluster_domain": "christy-pvc.openshift-on-power.com",
  "base_domain": "openshift-on-power.com",
```